src/main/java/org/betonquest/betonquest/objectives/MobKillObjective.java

```diff
@@ -68,6 +75,15 @@ public void onMobKill(final MobKilledEvent event) {
             }
         }
 
+        if (tags != null) {
+            final NBTEntity mcnbt = new NBTEntity(event.getEntity()); //Only for vanilla tags!
```

Comment:
We do not want comments anymore at all in new code. I think this comment is not necessary at all.